readme.md

```diff
@@ -1,6 +1,6 @@
 # RoslyJump
 
-RoslyJump is a free and open source extension for Visual Studio 2019 for mouseless code navigation. For now, it only has limited support of C# 8.0.
+RoslyJump is a free and open source extension for Visual Studio 2019/2022 for mouseless code navigation. For now, it only has limited support of C# 8.0.
```

Comment:
> Visual Studio 2019/2022

It should be extension version-agnostic because the same `readme.md` is published for both VS 2019 and VS 2022.